R/dataframe.R

```diff
@@ -229,6 +229,12 @@ as_tibble.default <- function(x, ...) {
 
 #' @export
 #' @rdname as_tibble
+as.tibble <- function(x, ...) {
```

Comment:
No problem, I'll change `as.tibble` alias to match `as_data_frame` alias.

Could you explain why you find it surprising that `UseMethod("as_tibble")` works? This way the newly created `as.tibble` generic dispatches to the methods of `as_tibble` generic. How would it be different from dispatching to the methods of `as_data_frame` generic?